### PR TITLE
Add missing error messages for sucker_punch, sneakers and qu

### PR DESCRIPTION
**Pull request by meinac**

### CODE REVIEW

**Comment by cristianbica** [comment]
The error message is a bit misleading the user is currently using a queueing backend (sneakers, sp, qu) but the backend does not support scheduling jobs. We should say: "This queueing backend doesn't not support scheduling jobs. To see what features are supported see http://api.rubyonrails.org/classes/ActiveJob/QueueAdapters.html"

**Comment by meinac** [comment]
@cristianbica thanks for the response, I've changed error messages.

File: activejob/lib/active_job/queue_adapters/qu_adapter.rb [hunk 1]
```diff
@@ -23,7 +23,7 @@ def enqueue(job, *args) #:nodoc:
       end
 
       def enqueue_at(job, timestamp, *args) #:nodoc:
-        raise NotImplementedError
+        raise NotImplementedError.new("This queueing backend doesn't not support scheduling jobs. To see what features are supported see http://api.rubyonrails.org/classes/ActiveJob/QueueAdapters.html")
```
Comment (senny) [inline activejob/lib/active_job/queue_adapters/qu_adapter.rb]: can we skip the `.new` and use this `raise` syntax:

```ruby
raise NotImplementedError, "message"
```

Did a quick grep and that's what most of Rails uses.

**Comment by meinac** [comment]
@senny I've also changed inline adapter raise error to use rails way.

File: activejob/lib/active_job/queue_adapters/qu_adapter.rb [hunk 2]
```diff
@@ -23,7 +23,7 @@ def enqueue(job, *args) #:nodoc:
       end
 
       def enqueue_at(job, timestamp, *args) #:nodoc:
-        raise NotImplementedError
+        raise NotImplementedError, "This queueing backend doesn't not support scheduling jobs. To see what features are supported see http://api.rubyonrails.org/classes/ActiveJob/QueueAdapters.html"
```
Comment (senny) [inline activejob/lib/active_job/queue_adapters/qu_adapter.rb]: either `does not` or `doesn't`. Let's go with `does not`.

Comment (senny) [inline activejob/lib/active_job/queue_adapters/qu_adapter.rb]: Let's reword: "To see what features are supported see" to "To see what features are supported go to"

**Comment by meinac** [comment]
@senny Thank you for your suggestions, I've fixed the message.

**Comment by senny** [comment]
@meinac thank you :yellow_heart:

**Pull request merged**